pkg/controller/rest/rest.go

```diff
@@ -59,7 +61,6 @@ func SendError(rw http.ResponseWriter, err command.Error) {
 // SendHTTPStatusError sends given http status code to response with error body
 func SendHTTPStatusError(rw http.ResponseWriter, httpStatus int, code command.Code, err error) {
 	rw.WriteHeader(httpStatus)
-	rw.Header().Set("Content-Type", "application/json")
```

Comment:
why is this removed?

Reply:
line #35 always gets called, so it gets set twice in case of errors